package.json

```diff
@@ -17,7 +17,7 @@
     "@types/koa-compose": "^3.2.5",
     "@types/koa-logger": "^3.1.1",
     "@types/koa__cors": "^2.2.3",
-    "@types/koa__router": "^8.0.1",
+    "@types/koa__router": "^8.0.2",
```

Comment:
weird naming convention, I also found the oldest `@types/koa-router` which is probably abandoned

Reply:
Double underscore is for the namespaced convention:

- `@types/koa-router` is for `koa-router`
- `@types/koa__router` is for `@koa/router`